beast-gtk/bsttrackview.hh

```diff
@@ -20,11 +20,13 @@ typedef	struct	_BstTrackView	   BstTrackView;
 typedef	struct	_BstTrackViewClass BstTrackViewClass;
 struct _BstTrackView
 {
-  BstItemView	          parent_object;
-  Bse::SongS              song;
-  BstTrackRoll	         *troll;
-  BstTrackRollController *tctrl;
-  GtkWidget		 *repeat_toggle;
+  BstItemView                     parent_object;
+  Bse::SongS                      song;
+  std::map<SfiProxy, Bse::TrackS> track_map;
```

Comment:
Note, I'm perfectly happy with adding type aliases for complex types, so you don't need to reformat all members. E.g.

    using TrackMap = std::map<SfiProxy, Bse::TrackS>;
    // other members...
    TrackMap    track_map;